### ISSUE TITLE
Expose server (manual port) and secure connections

### ISSUE THREAD

**Issue by psyciknz** [UnicornTranscoder/UnicornLoadBalancer#20]
The ports for remote access confuse me.

I use a non standard (2530).  On my router this is a forward from 2530  to my plex machine on 32400.

To get load balancer - which seems to run on 3001 to work with remote access, I'd have thought setting it all up so the load balancer was on 32400, port forwarded from 2350 (router), and put plex on 32401.

I can get a connect via the load balancer, but can't get the secure connections to work.

**Comment by psyciknz** [comment]
So I followed this really great post: https://forums.plex.tv/t/release-unicorntranscoder-create-a-plex-transcoding-cluster/281679/68  and got a working local setup using my domain name, through caddy to the load balancer.

For the life of me, I couldnt' get external access working, either via plex.tv or using my domain name.  I kept getting a S1002 error.

I couldn't see any logging (I'm using the docker containers https://github.com/magn2o/unicorn-docker for master and slave.

LB Port 3001
Plex port 32400
Advertise Ip is my https://plex.mydomain.com:443
